.mergify.yml

```diff
@@ -0,0 +1,100 @@
+pull_request_rules:
+  - name: automatic merge for Dependabot pull request that pass CI
+    conditions:
+      - author=dependabot[bot]
+      - check-success=build-workflow-complete
```

Comment:
this can be removed; DP won't merge unless all checks pass

Reply:
👍 removed.